test/spec/SearchResults/getFacetValues-facetOrdering.js

```diff
@@ -0,0 +1,1518 @@
+'use strict';
+
+var SearchResults = require('../../../src/SearchResults');
+var SearchParameters = require('../../../src/SearchParameters');
+
+describe('disjunctive facet', function() {
+  test.each([
+    [
+      'nothing ordered',
```

Comment:
Tests and logic are already complex so I think we should add the expectations in the test names:

```suggestion
      'sorts by count by default',
```

Wdyt?